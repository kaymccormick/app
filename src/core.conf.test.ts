const configJs = require('../core.conf');

test('configJs', () => {
expect(configJs).toMatchSnapshot();
});
