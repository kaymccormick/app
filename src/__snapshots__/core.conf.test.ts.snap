// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configJs 1`] = `
Object {
  "modules": Array [
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
    [Function],
  ],
}
`;
